source/includes/inkling-reference/_schema.html.md

```diff
@@ -108,102 +120,149 @@ schema MNIST_training_data_schema
 end
 ```
 
+Select the Inkling tab to show a schema that has a field with a primitive type and a field with a structured type.
+
+#inkling_types
```

Comment:
You've created a new heading here by doing this when actually you just want to link to the heading that already exists (the line below this).